CountryPickeriOS13+/CountryPickerSPM/Sources/CountryPickerSPM/CountryPickerWithSectionViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CountryPickerWithSectionViewModel.swift
+//  
+//
+//  Created by Surya on 09/07/23.
+//
+
+import Combine
+
+class CountryPickerWithSectionViewModel: ObservableObject {
+    
+    @Published var sections: [Section] = []
+    @Published var selectedCountry: Country?
+    
+    private let dataService: any CountryListDataSource
+    private let mapper: SectionMapper
+    
+    internal init(dataService: any CountryListDataSource,
+                  favoriteCountriesLocaleIdentifiers: [String],
+                  mapper: SectionMapper
+    ) {
+        self.dataService = dataService
+        self.mapper = mapper
+        self.selectedCountry = dataService.lastCountrySelected
+        self.selectedCountry = selectedCountry
+        defer {
```

Comment:
Adding defer does not make any sense as we are running all the code synchronously.